docs/stores_api.md

```diff
@@ -0,0 +1,68 @@
+# Stores API
+
+## The Store object
+
+Attributes:
+
+- `location_type`: Whether the location is a Store or a Warehouse
+- `long_name`: Full store name
+- `name`: Store name
+- `opening_time`: See [opening_time](stores_api.md#opening_time)
+- `phone`: Store phone number; phone number for Express stores goes to Best Buy Customer Service
+- `services`: Collection of services offered at each Best Buy store
+- `store_id`: Store number
+- `store_type`: Indicates the type of store, and is only present if locationType is ‘Store’. See the [official documentation](https://bestbuyapis.github.io/api-documentation/#common-attributes56) for more details
+- `store_address`: See [The Address object](stores_api.md#the-address-object)
+
+#### `opening_time`
+
+Array of hashes containing info about the opening hours for each day for the next two weeks. It has the following keys:
+
+- `:day`: Name of the day of the week (e.g.: 'Sunday')
+- `:date`: Date of the day (YYYY-MM-DD)
+- `:open`: Opening time (24-hour clock) (`'Close'` if the store is closed)
+- `:close`: Closing time (24-hour clock) (`'Close'` if the store is closed)
```

Comment:
What do these return? A string with the time? Like `"17:00"` for instance?

Reply:
Yes! I'll add it to the docs